data/_nav.yml

```diff
@@ -215,23 +221,30 @@ reference:
         - url: /docs/reference/experimental.html
           title: Experimental API Markers
 
-    - title: Core Libraries
+    - title: Core Libraries Reference
       content:
         - url: /api/latest/jvm/stdlib/index.html
           title: Standard Library
         - url: /api/latest/kotlin.test/index.html
           title: kotlin.test
 
-    - title: Reference
+    - title: Language Reference
```

Comment:
The contents of this group are not what's conventionally called a language reference. This is another problem of discoverability. Please propose solutions to it